chapter-06/inprogress/suggestions.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import "sort"
+
+// bookSuggestions are the suggested books for a given book.
+// Suggestions are the neighbour-books on the shelves.
+type bookSuggestions map[Book]bookCollection
```

Comment:
There are quite a few types having a prefix `book`, you probably can make the code shorter by introducing a `book` package and dropping all such prefixes.

Reply:
This chapter is going to be chapter 3 and we did not want to introduce the package yet but rather have everything at the same level to simplify the chapter's content.

Reply:
In that case, actually, if the main is going to be small... then it should be possible to drop them nevertheless in many cases.